chebi-update/src/test/java/org/reactome/release/chebiupdate/JavaVsPerlComparison.java

```diff
@@ -18,58 +18,82 @@
 		{
 			StringBuilder sb = new StringBuilder();
 			GKInstance perlInstance = perlUpdatedDB.fetchInstance(javaInstance.getDBID());
-			
+
 			// This is a filter, to filter instances out of the comparison, based on attributes.
 			//
-			// This filter excludes anything where the attribute is one of: name, displayName, referenceEntity, formula, identifier
-			// AND the object in question is a SimpleEntity and the attribute is referenceEntity.
+			// This filter excludes anything where the attribute is one of: name, displayName,
+			// referenceEntity, formula, identifier AND the object in question is a SimpleEntity and the attribute
+			// is referenceEntity.
 			Predicate<? super SchemaAttribute> customAttributeNameFilter = schemaAttribute -> {
 				// Is the attribute's name in the list?
-				boolean nameIsInList = (new HashSet<String>(Arrays.asList(ReactomeJavaConstants.name, ReactomeJavaConstants._displayName, ReactomeJavaConstants.referenceEntity, ReactomeJavaConstants.formula, ReactomeJavaConstants.identifier))).contains(schemaAttribute.getName());
-				
-				// This condition below is to avoid recursing from the SimpleEntity back the ReferenceMolecule via SimpleEntity's referenceEntity attribute.
-				// The maxRecursionDepth would prevent things from getting too out of hand, but you'll still get a lot of redundant diffs if you don't do this.
+				boolean nameIsInList = (new HashSet<>(
+					Arrays.asList(
+						ReactomeJavaConstants.name, ReactomeJavaConstants._displayName,
+						ReactomeJavaConstants.referenceEntity, ReactomeJavaConstants.formula,
+						ReactomeJavaConstants.identifier
+					)
+				)).contains(schemaAttribute.getName());
+
+				// This condition below is to avoid recursing from the SimpleEntity back the ReferenceMolecule via
+				// SimpleEntity's referenceEntity attribute.  The maxRecursionDepth would prevent things from getting
+				// too out of hand, but you'll still get a lot of redundant diffs if you don't do this.
 				// Is the attribute "referenceEntity" and is it on a SimpleEntity object?
 				@SuppressWarnings("unchecked")
-				boolean simpleEntityWithReferenceEntityAttribute = ( new ArrayList<SchemaClass>(schemaAttribute.getSchemaClass())).get(0).getName().equals(ReactomeJavaConstants.SimpleEntity)
-					&& schemaAttribute.getName().equals(ReactomeJavaConstants.referenceEntity);
-				
+				boolean simpleEntityWithReferenceEntityAttribute =
+					(new ArrayList<SchemaClass>(
+						schemaAttribute.getSchemaClass()
+					)).get(0).getName().equals(ReactomeJavaConstants.SimpleEntity) &&
+					schemaAttribute.getName().equals(ReactomeJavaConstants.referenceEntity);
+
 				return nameIsInList && !simpleEntityWithReferenceEntityAttribute;
 			};
-			
-			int currentDiff = DBObjectComparer.compareInstances(javaInstance, perlInstance, sb, 1, customAttributeNameFilter, true);
-			
+
+			int currentDiff = DBObjectComparer.compareInstances(
+				javaInstance, perlInstance, sb, 1, customAttributeNameFilter, true
```

Comment:
What does the `1` mean in `compareInstances`? If it is an internal method, is it worth having it as a named variable?

Reply:
Done: https://github.com/reactome/data-release-pipeline/pull/121/commits/19a51ca5c32d5b1f22091bcfdfdf0bcf58690153